src/Test/Utilities/Portable/Assert/AssertEx.cs

```diff
@@ -12,6 +12,7 @@
 using DiffPlex;
 using DiffPlex.DiffBuilder;
 using DiffPlex.DiffBuilder.Model;
+using Microsoft.CodeAnalysis;
```

Comment:
Is this using necessary?

Reply:
No. Seems to be a hold over from an intermediate change. Will delete.